libs/js-sdk/src/update-strategies/createPollingUpdateStrategy.ts

```diff
@@ -66,13 +64,19 @@ export function createPollingUpdateStrategy(
         onRequest() {
             return undefined
         },
-        updateAudiences(state, updatedAudiences) {
-            currentAudiences = updatedAudiences
+        async updateAudiences(state, updatedAudiences) {
```

Comment:
Actually, looking at this, because we have exposed update audiences on state, we could remove this function from the update strategies entirely.

Although might have to call `lastModified = undefined` in connect?

Don't mind leaving it here, but it looks identical to the last one.